dumper-integration-tests/redshift-tests/src/main/java/com/google/edwmigration/dumper/csv/CsvUtil.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.csv;
+
+import static java.lang.Integer.parseInt;
+
+/** A helper class for reading and extracting data from CSV files. */
+public final class CsvUtil {
+
+  private CsvUtil() {}
+
+  /**
+   * @return String or an empty string if null.
+   */
+  public static String getStringNotNull(String value) {
+    return value == null ? "" : value;
+  }
```

Comment:
This is the same as `Strings.nullToEmpty(value)` ( `com.google.common.base.Strings` )

Reply:
Also Guava MoreObjects.firstNonNull or java.util.Objects.firstNonNull

Reply:
This method is no longer needed and it's removed, but I noted all three methods your guys suggested, thanks, good to know.

Done.